misc/multistream-select/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.8.4 [2020-10-20]
+
+- Temporarily disable the internal selection of "parallel" protocol
+  negotiation for the dialer to later change the response format of the "ls"
+  message for spec compliance.
```

Comment:
```suggestion
  message for spec compliance. See https://github.com/libp2p/rust-libp2p/issues/1795.
```

For those only reading the changelog, I think citing the issue for details is a good idea.